docs(borealis): initial borealis CLI docs DOC-456 (#918)
armory/docs: content/en/borealis/borealis-requirements.md

To use Project Aurora/Borealis, make sure you meet the following requirements.

## Armory Remote Network Agent

> Project Aurora/Borealis uses agents that run in target Kubernetes clusters to communicate with Armory hosted cloud services, specifically the Agent Hub. Make sure the [networking](#networking) requirements are met so that the agents can communicate with the Agent Hub. 

Reviewer: ```suggestion
Project Aurora/Borealis uses agents that run in target Kubernetes clusters to communicate with Armory hosted cloud services, specifically the Agent Hub. Make sure your environment meets the [networking](#networking) requirements so that the agents can communicate with the Agent Hub. 
```

> There are no additional requirements for installing the agents that Project Aurora/Borealis use. For information about how to install these agents (known as the Remote Networking Agent), see [Enable Project Aurora in target Kubernetes clusters]({{< ref "aurora-install#enable-aurora-in-target-kubernetes-clusters" >}}).

Reviewer: ```suggestion
There are no additional requirements for installing the agents that Project Aurora/Borealis use. For information about how to install these agents, see [Enable Project Aurora in target Kubernetes clusters]({{< ref "aurora-install#enable-aurora-in-target-kubernetes-clusters" >}}).
```
